client/app/pages/dashboards/components/DashboardListEmptyState.tsx

```diff
@@ -21,6 +21,8 @@ export default function DashboardListEmptyState({ page, searchTerm, selectedTags
   switch (page) {
     case "favorites":
       return <BigMessage message="Mark dashboards as Favorite to list them here." icon="fa-star" />;
+    case "my":
+      return <BigMessage message="Create your own dashboards to list them here." icon="fa-line-chart" />;
```

Comment:
Maybe make it more like the queries one? https://github.com/getredash/redash/blob/04edf16ed42cdf9d005e7a512162cee7ca298c9c/client/app/pages/queries-list/QueriesListEmptyState.jsx#L21

Reply:
I considered that. The queries page has a nice link to create a new query, but we use a different mechanism to create a new dashboard (popping a modal dialog to name it first), so I couldn't just copy the existing code. I could dig into how to make this work if you think it's important.

Reply:
@justint-db All you need is this (see click handler; `onMouseUp` is used here because that's menu item, in your case use just regular `onClick`):

https://github.com/getredash/redash/blob/04edf16ed42cdf9d005e7a512162cee7ca298c9c/client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx#L125-L127

Reply:
Thanks! Done.

Reply:
See updated image above.